service/commentutil/commentutil.go

```diff
@@ -77,6 +77,35 @@ func MarkdownComment(c *reviewdog.Comment) string {
 	return sb.String()
 }
 
+// MarkdownSuggestions creates diff in markdown for suggested changes
```

Comment:
Can you add a link to the GitLab document?
Maybe https://docs.gitlab.com/ee/user/discussions/#multi-line-suggestions